tests/Admin/Admin_Page_Tests.php

```diff
@@ -63,4 +64,24 @@ public function test_render_page() {
 		$this->assertStringContainsString( ' value="Check it!"', $output );
 		$this->assertStringNotContainsString( plugin_basename( WP_PLUGIN_CHECK_MAIN_FILE ), $output );
 	}
+
+	public function test_filter_plugin_action_links() {
+
+		$action_links = $this->admin_page->filter_plugin_action_links( array(), 'test-plugin/test-plugin.php' );
+
+		$this->assertEmpty( $action_links );
+
+		$base_file = plugin_basename( WP_PLUGIN_CHECK_MAIN_FILE );
+
+		$action_links = $this->admin_page->filter_plugin_action_links( array(), $base_file );
+
+		$this->assertEquals(
+			sprintf(
+				'<a href="%1$s">%2$s</a>',
+				esc_url( admin_url() . 'tools.php?page=plugin-check&plugin=' . $base_file ),
+				esc_html__( 'Check this plugin', 'plugin-check' )
+			),
+			$action_links[0]
+		);
+	}
```

Comment:
See my comment above: You can simplify this test since the link should appear on every plugin, even when it is not active.

Reply:
Done